### ISSUE TITLE
Multi scheme migrations issue.

### ISSUE THREAD

**Issue by Mark1Z** [mikro-orm/mikro-orm#1143]
**Describe the bug**
Migrations do not respect table scheme.

**Stack trace**
```
TableExistsException: create table "test"."DEVICES" ("ID" serial primary key, "TOKEN" varchar(255) not null); - relation "DEVICES" already exists
    at PostgreSqlExceptionConverter.convertException (/home/virchenko/Projects/phyapi/node_modules/@mikro-orm/postgresql/PostgreSqlExceptionConverter.js:38:24)
    at PostgreSqlDriver.convertException (/home/virchenko/Projects/phyapi/node_modules/@mikro-orm/core/drivers/DatabaseDriver.js:171:54)
    at /home/virchenko/Projects/phyapi/node_modules/@mikro-orm/core/drivers/DatabaseDriver.js:175:24
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at Function.runSerial (/home/virchenko/Projects/phyapi/node_modules/@mikro-orm/core/utils/Utils.js:458:22)
    at MigrationRunner.run (/home/virchenko/Projects/phyapi/node_modules/@mikro-orm/migrations/MigrationRunner.js:17:13)
```

**To Reproduce**
1. Create an entity, for example:
```
import { Entity, PrimaryKey, Property } from '@mikro-orm/core';

@Entity({
    tableName: 'test.DEVICES',
})
export class Device {
    @PrimaryKey({ fieldName: 'ID', type: 'number' })
    id!: number;

    @Property({ fieldName: 'TOKEN' })
    token!: string;
}
```
2. Generate a migration:
```
npx mikro-orm migration:create
```
where output will be like this:
```
import { Migration } from '@mikro-orm/migrations';

export class Migration20201201130518 extends Migration {

  async up(): Promise<void> {
    this.addSql('create table "test"."DEVICES" ("ID" serial primary key, "TOKEN" varchar(255) not null);');
  }

}
```
3. Apply the migration:
```
npx mikro-orm migration:up
```
4. Generate a new migration:
```
npx mikro-orm migration:create
```
where output will be the same:
```
import { Migration } from '@mikro-orm/migrations';

export class Migration20201201130522 extends Migration {

  async up(): Promise<void> {
    this.addSql('create table "test"."DEVICES" ("ID" serial primary key, "TOKEN" varchar(255) not null);');
  }

}
```
5. Apply the migration:
```
npx mikro-orm migration:up
```

**Expected behavior**
Correct migrations: 
- first migration with "create scheme 'test';" statement and down() method;
- second migration must be empty.

**Versions**

| Dependency | Version |
| - | - |
| node | v12.20.0 |
| typescript | 4.0.5 |
| mikro-orm | 4.3.0 |
| PostgreSQL | 13.1 |

**Comment by B4nan** [comment]
Down migrations are planned, but that will require deeper refactoring of how the schema diffing works. I marked this as a bug as the second migration should be indeed empty. Will close this with a fix for that - the rest are feature proposals, not bugs - schema diffing is not implemented at all, just like down migrations. Feel free to create new issues/feature requests for that, they should be separate as those are distinct features.

**Comment by B4nan** [comment]
Fixed in `4.3.3-dev.12`
